rosbridge_server/src/rosbridge_server/websocket_handler.py

```diff
@@ -126,3 +127,11 @@ def send_message(self, message):
 
     def check_origin(self, origin):
         return True
+
+    def get_compression_options(self):
+        cls = self.__class__
+
+        if not cls.use_compression:
+            return None
+
+        return {}
```

Comment:
Do we want to document the behaviour of this? At least a copy paste from [Tornado's docs](https://www.tornadoweb.org/en/stable/websocket.html#tornado.websocket.WebSocketHandler.get_compression_options) can help:

> If this method returns None (the default), compression will be disabled. If it returns a dict (even an empty one), it will be enabled. 

Should we expose the [compression and memory](https://docs.python.org/3.6/library/zlib.html#zlib.compressobj) level as well? IMO at least a comment describing the defaults would suffice.

Reply:
I didn't see any of the websocket server's params documented anywhere, I'm happy to add a note if you can point me at a place to leave it.  I assume the Tornado docs will be the first point of reference for anybody looking at this.

Compression and memory level only became available in Tornado 4.5, so if we exposed them it might take extra work to support kinetic.  I didn't feel like adjusting those settings would be particularly valuable.